src/components/CCIP/ChainHero/ChainHero.css

```diff
@@ -54,19 +54,20 @@
 
 .ccip-chain-hero__top {
   display: flex;
+  flex-direction: column-reverse;
```

Comment:
Not sure if it is this page, but side padding still too small here:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/c14544f7-894f-4ce1-88d5-c5e228336af7">